### ISSUE TITLE
cli: update 'popper ci' services configuration to install popper via pip

### ISSUE THREAD

**Issue by ivotron** [systemslab/popper#583]
all the CI configurations install popper using the "manual" method of obtaining dependencies by cloning git submodules in the repo. This method will be deprecated when #563 gets merged. So we need to update so it does:

```bash
git clone https://github.com/systemslab/popper
pip install popper/cli
```

**Comment by JayjeetAtGithub** [comment]
@ivotron I would like to send a PR on this.

**Comment by ivotron** [comment]
Reopening this as I found an issue. The correct commands should be:

```bash
git clone https://github.com/systemslab/popper /tmp/popper
pip install /tmp/popper/cli
```

otherwise the tests for the popper project also will be executed on the project.

**Comment by JayjeetAtGithub** [comment]
@ivotron This looks interesting! I can't reproduce this error. Can you please help me?

**Comment by ivotron** [comment]
this is not an error that manifests on this repo but on others using
popper. For example this: https://github.com/popperized/geni

**Comment by ivotron** [comment]
closed by cce047f
